site/content/contribute/getting-started/test-policy.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Test Policy"
```

Comment:
How do we all feel about the term "policy" vs "guideline"? It may be minor and I'm definitely not 5/5 on avoiding the term policy when we can, but I find the term "guidelines" preferable.

Reply:
The original ask was to have a "unit test policy" but I honestly don't think I was able to deliver such expectation. It even expanded (in a different direction) into a broader scope which included end-to-end test.
I recently mentioned this uncertainty during our 1:1 but I'm not sure what's the best term to use given the content of the document.

Reply:
Per offline discussion, it's agreed to change from policy to guideline.

Now updated.